src/com/codefortomorrow/intermediate/chapter10/practice/BinaryReader.java

```diff
@@ -0,0 +1,30 @@
+package com.codefortomorrow.intermediate.chapter10.practice;
+
+/*
+    Create a program called BinaryReader in which a 4 x 4
+    array of booleans is created. Have the computer loop
+    through the array, printing a 0 if the current element is
+    set to false and a 1 if it set to true.

```

Comment:
Instead of having them create an array themselves, you might want to consider having this be a method instead (the 4x4 array is the parameter and the method either prints or returns the string)